src/dbnode/storage/flush.go

```diff
@@ -63,6 +64,7 @@ type flushManager struct {
 	// are used for emitting granular gauges.
 	state           flushManagerState
 	isFlushing      tally.Gauge
+	isColdFlushing  tally.Gauge
```

Comment:
Super nit, I wonder if we should call these a "compaction" flush instead of cold flush?  I'm not really too fussed, but interested to hear thoughts cc @justinjc @prateek @richardartoul

Reply:
(Like still keep the hot/cold terminology but just call these flushes "compaction flushes" because then if repairs uses this code path, it's not just cold writes that are being persisted per-se).

Reply:
Second thoughts, seems to hard to rename at this point, let's not bother.

Reply:
I actually renamed this from compact to cold flush midway through 😅. Maybe we'll re-tackle this when the usage of cold flush differs too much later.